Demo-Android/Gutenberg/src/main/java/org/wordpress/gutenberg/GutenbergView.kt

```diff
@@ -94,15 +91,6 @@ class GutenbergView : WebView {
                 view: WebView?,
                 request: WebResourceRequest?
             ): WebResourceResponse? {
-                if (!hasSetEditorConfig && withConfig) {
-                    handler.post {
-                        var editorInitialConfig = getEditorConfiguration()
-                        view?.evaluateJavascript(editorInitialConfig, null)
-
-                    }
-                    hasSetEditorConfig = true
-                }
```

Comment:
There is a race condition that results in this approach not consistently load the global values before the HTML page loaded. 

Unfortunately, Android does not provide a method similar to iOS' `addUserScript`. so this "push" approach was replaced with a "pull" approach on Android to ensure the value is available before the first WebView requests are made.